website/vercel.json

```diff
@@ -1205,7 +1205,46 @@
     {
       "source": "/getting-started/setup/instance-configuration/authentication/openid-connect-oidc/active-directory",
       "destination": "/getting-started/setup/instance-configuration/authentication/openid-connect-oidc/entra-id"
+    },
+	{
+	"source": "/widget-reference/file-picker",
+      "destination": "/reference/widgets/filepicker",
+      "permanent": true
+    },
+    {
+      "source": "/widget-reference/map",
+      "destination": "/reference/widgets/maps",
+      "permanent": true
+    },
+    {
+      "source": "/widget-reference/dropdown",
+      "destination": "/reference/widgets/select",
+      "permanent": true
+    },
+    {
+      "source": "/reference/datasources/querying-mongodb/whitelist-mongodb-cloud",
+      "destination": "https://studio3t.com/knowledge-base/articles/mongodb-atlas-login-ip-whitelisting/#how-to-whitelist-ip-addresses-on-mongodb-atlas",
+      "permanent": true
+    },
+    {
+      "source": "/write-code/reference/core-concepts/writing-code/ext-libraries",
```

Comment:
This is a wrong url. Let's deindex it, unless we really need a redirect for this.

Reply:
Removed and deindexed